monitors/index.js

```diff
@@ -124,13 +123,9 @@ const Poll = async function(callback) {
         "Bad environment variables! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party."
       );
     }
-    // TODO: Remove this price flag once we have built the pricefeed module.
-    if (!process.env.PRICE) {
```

Comment:
price env variable was still being passed through to the bot. this is clearly not needed any more.